neighborhood_data/update_data.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Run the scripts in this directory to regenerate data and copy it into the app.
+
```

Comment:
It would be good to add a `set -e` line here so that this script exits if any of its commands terminate abnormally.